ShippingApp/src/main/java/com/javaschool/queue/ConfigRabbit.java

```diff
@@ -0,0 +1,37 @@
+package com.javaschool.queue;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigRabbit {
+
+    @Value("${config-rabbit.queue}")
+    public String QUEUE_SHIPPING;
+
+    @Value("${config-rabbit.exchange}")
+    public String EXCHANGE_SHIPPING;
+
+    @Value("${config-rabbit.routing-key}")
+    public String ROUTING_KEY_SHIPPING;
+
+    @Bean
+    public Queue shippingQueue() {
+        return new Queue(QUEUE_SHIPPING, false);
```

Comment:
Why are you creating a `Queue`, `TopicExchange` and a `Binding` if you are only consuming out queue?
With this, you are creating a new one on each time you run your application, AFAIK

Reply:
It is not necessary, it is removed.